lxd/resource_lxd_container_test.go

```diff
@@ -660,7 +660,7 @@ resource "lxd_container" "container1" {
 
   file {
     content = "Hello, World!\n"
-    target_file = "/tmp/foo/bar.txt"
+    target_file = "/foo/bar.txt"
```

Comment:
was this the cause of the test failures before? 
does `/tmp` not exist before container start?